servers/src/test/java/tachyon/master/file/FileSystemMasterPermissionCheckTest.java

```diff
@@ -468,6 +471,174 @@ public void readDirFailTest() throws Exception {
     verifyGetFileId(TEST_USER_2, file);
   }
 
+  @Test
+  public void setStateSuccessTest() throws Exception {
+    // set unmask
+    TachyonConf conf = MasterContext.getConf();
+    conf.set(Constants.SECURITY_AUTHORIZATION_PERMISSIONS_UMASK, "044");
+    MasterContext.reset(conf);
+
+    String file = TEST_DIR_URI + "/testState1";
+    verifyCreate(TEST_USER_1, file, false);
+    SetStateOptions expect = getRamdomSetState();
```

Comment:
in general, we avoid introduce randomness in unit tests, as test failures are not surely reproducible. Can you simply pick ttl/recursive flag here as they seem not critical here